.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,52 @@
+---
+name: Bug report
+about: Report a problem with this library that needs to be fixed.
+title: ''
+labels: triage
+assignees: ''
+
+---
+
+**Before filing this issue**
```

Comment:
I wonder if this should be formatted differently from the rest of the template to better delineate between guidance text and text that is supposed to be replaced. Maybe the instructions under "Describe the bug" onward could have all but the titles commented out? e.g.

**Describe the bug**
<!-- A clear and concise description of what the bug is. -->

Reply:
+1

Reply:
Definitely a good call - I added comments for these sections and also wrapped the big caveat at the top of the page in a comment so that none of these will be visible in the rendered markdown.